packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -93,6 +94,9 @@ exports.sourceNodes = async (
   // Default concurrentFileRequests to `20`
   concurrentFileRequests = concurrentFileRequests || 20
 
+  // Default skipFileDownloads to false.
+  skipFileDownloads = skipFileDownloads || false
+
```

Comment:
Nit: Above, you could set the default

```js
let {
  skipFileDownloads = false,
} = pluginOptions
```

Reply:
(I guess no need to change it here, for the sake of consistency, but it could be cleaned up later perhaps)